test/functional/feature_assumeutxo.py

```diff
@@ -205,6 +208,35 @@ def run_test(self):
         assert_equal(loaded['coins_loaded'], SNAPSHOT_BASE_HEIGHT)
         assert_equal(loaded['base_height'], SNAPSHOT_BASE_HEIGHT)
 
+        # After loading the snapshot, check nTx values in the starting block
+        # before loading the snapshot, the first block after it, the last block
+        # before the snapshot block, and the snapshot block.
+        start_hash = n1.getblockhash(height=START_HEIGHT)
+        start_next_hash = n1.getblockhash(height=START_HEIGHT+1)
+        snapshot_prev_hash = n1.getblockhash(height=SNAPSHOT_BASE_HEIGHT-1)
+        snapshot_hash = n1.getblockhash(height=SNAPSHOT_BASE_HEIGHT)
+        assert_equal(n1.getblockheader(start_hash)["nTx"], 1)
+        assert_equal(n1.getblockheader(start_next_hash)["nTx"], 1)
+        assert_equal(n1.getblockheader(snapshot_prev_hash)["nTx"], 1)
+        assert_equal(n1.getblockheader(snapshot_hash)["nTx"], 1)
+
+        # After loading the snapshot, check nChainTx values indirectly, by using the
```

Comment:
> Why check these values indirectly instead of directly (field `txcount`)?

Wow, I did not notice that field 🤦. That will make the test a lot better.